src/Common/src/System/Marvin.cs

```diff
@@ -110,9 +110,9 @@ private static ulong GenerateSeed()
         {
             using (RandomNumberGenerator rng = RandomNumberGenerator.Create())
             {
-                var bytes = new byte[sizeof(ulong)];
+                Span<byte> bytes = stackalloc byte[sizeof(ulong)];
                 rng.GetBytes(bytes);
-                return BitConverter.ToUInt64(bytes, 0);
+                return MemoryMarshal.Cast<byte, ulong>(bytes)[0];
```

Comment:
This is not portable to platforms with strong alignment requirements. This should be `MemoryMarshal.Read<ulong>(bytes)` instead.